VideoLocker/src/main/java/org/edx/mobile/model/api/TranscriptModel.java

```diff
@@ -69,4 +69,25 @@
         String label = context.getResources().getString(resId);
         return label;
     }*/
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+
+        TranscriptModel that = (TranscriptModel) o;
+
+        if (englishUrl != null ? !englishUrl.equals(that.englishUrl) : that.englishUrl != null)
+            return false;
+        if (spanishUrl != null ? !spanishUrl.equals(that.spanishUrl) : that.spanishUrl != null)
+            return false;
+        if (germanUrl != null ? !germanUrl.equals(that.germanUrl) : that.germanUrl != null)
+            return false;
+        if (portugueseUrl != null ? !portugueseUrl.equals(that.portugueseUrl) : that.portugueseUrl != null)
+            return false;
+        if (chineseUrl != null ? !chineseUrl.equals(that.chineseUrl) : that.chineseUrl != null)
+            return false;
+        return frenchUrl != null ? frenchUrl.equals(that.frenchUrl) : that.frenchUrl == null;
```

Comment:
This can be shortened by using the `TextUtils.equals()` utility. These could also all be joined together like `return TextUtils.equals(englishUrl, that. englishUrl) && TextUtils.equals(spanishUrl, that. spanishUrl)` etc.